.deployment/barista/Jenkinsfile

```diff
@@ -64,28 +64,49 @@ pipeline {
                 writeFile(file: "${ENVIRONMENT_FILE}",  text: text)
               }
             }
-
-            dir(WORKSPACE_DIR) {
-              sh '''
-                rm -rf ./node_modules/@dynatrace/barista-icons
-                cp -R ./dist/tmp/barista-icons ./node_modules/@dynatrace/barista-icons
-              '''
-            }
           }
         }
 
         stage("Prepare Workspace") {
           steps {
+
+            script {
+              if (!(params.INTERNAL_VERSION ==~ /(?i)(Y|YES|T|TRUE|ON|RUN)/ )) {
```

Comment:
:D was this due to all possible values coming from somewhere?

Reply:
no previously i had the public icons in the node_modules and I changed that so the internal are default so I need the check here